assessments/GOLD/earthmover.yaml

```diff
@@ -0,0 +1,176 @@
+version: 2
+
+
+config:
+  log_level: INFO
+  output_dir: ${OUTPUT_DIR}
+  memory_limit: 1GB
+  state_file: ${STATE_FILE}
+  show_graph: False
+  show_stacktrace: true
+  parameter_defaults:
+    STUDENT_ID_NAME: 'edFi_studentUniqueID' # default to the column added by the apply_xwalk package of student ID xwalking feature
+    POSSIBLE_STUDENT_ID_COLUMNS: 'ChildID,SourceChildID,Identifier,Student ID#'
+
+sources:
+  assessments:
+    file: ./seeds/assessments.csv
+    header_rows: 1
+  assessmentReportingMethodDescriptors:
+    file: ./seeds/assessmentReportingMethodDescriptors.csv
+    header_rows: 1
+  performanceLevelDescriptors:
+    file: ./seeds/performanceLevelDescriptors.csv
+    header_rows: 1
+  gradeLevelDescriptors:
+    file: ./seeds/gradeLevelDescriptors.csv
+    header_rows: 1
+
+  input:
+    # THIS FILE DOES NOT EXIST; IT MUST BE SUPPLIED BY THE USER!
+    file: ${INPUT_FILE}
+    header_rows: 1
+    # See the accompanying bundle_metadata.json for a list of required columns for this file
+    # Or see data/sample_anonymized_file.csv
+
+
+{% set assessments = {
+  'Social-Emotional':'GOLD_SE',
+  'Physical': 'GOLD_PHY',
+  'Language': 'GOLD_LANG',
+  'Literacy': 'GOLD_LIT',
+  'Spanish Language': 'GOLD_LANG_Spanish',
+  'Spanish Literacy': 'GOLD_LIT_Spanish',
+  'Cognitive': 'GOLD_COG',
+  'Mathematics':'GOLD_MATH'
+}
+%}
+
+{% set school_years = [
+  'Fall 2022/2023'
```

Comment:
We are going to have other files that have different values here (both for season & year), is there a way we could make this dynamic?

Reply:
I have refactored the code to make this and the below `admin_date` to be more dynamic. However, its validity depends on how we are assuming source files are managed.